packages/nocms-plugin-file-resources/test/index.spec.js

```diff
@@ -0,0 +1,136 @@
+import fs from 'fs';
+import { promisify } from 'util';
+import { expect } from 'chai';
+import { activate } from '../src/index.js';
+
+const readFileAsync = promisify(fs.readFile);
+
+let resourceProviders = [];
+
+let fakeFiles = [];
+let writtenFiles = [];
+
+let filePaths = [
+	'test/test-files/fakeJSFileName.js',
+	'test/test-files/fakeHTMLFileName.html',
+	'test/test-files/fakeExtFileName.fakeExtension'
+];
```

Comment:
I think this belongs in the initial describe scope (line 45-ish).

Reply:
moved